### DISCUSSION TITLE
EU/EBSI Trust Lists

### DISCUSSION THREAD

**Discussion by trbouma** [trustoverip/tswg-trust-registry-tf#73] [General]
A good article by Andy Tobin

https://www.linkedin.com/pulse/eu-wallet-depth-1-trusted-lists-andrew-tobin/

Referencing the EBSI Issuers Trust Model

https://ec.europa.eu/digital-building-blocks/wikis/display/EBSIDOC/Issuers+trust+model+-+Accreditation+of+Issuers

Are we solving for the same thing?

**Comment by andorsk** [comment]
Thanks @trbouma . Sorry I didn't see this prior to the TRTF meeting. This is a good question.
